stablehlo/transforms/StablehloLegalizeQuantToInt.cpp

```diff
@@ -1247,6 +1247,18 @@ class ConvertGenericOp : public ConversionPattern {
       return failure();
     }
 
+    if (isa<stablehlo::MinOp, stablehlo::MaxOp>(op)) {
```

Comment:
That this point of code we know that Max/Min op is illegal, which means one of the operands or results has a quantized type. 


Together with min/max constraints

```
`lhs` | tensor or per-tensor quantized tensor
`rhs` | tensor or per-tensor quantized tensor 
`result` | tensor or per-tensor quantized tensor 
`baseline_type(lhs) = baseline_type(rhs) = baseline_type(result)`.
```
we know that the type of lhs/rhs/result should be per-tensor quantized.

Reply:
Maybe make this a code comment.

```
// Min/max only support per-tensor quantization.
```

Reply:
done.